Core/Base/AgentModule.py

```diff
@@ -141,7 +141,7 @@ def __init__( self, agentName, loadName, baseAgentName = False, properties = {}
     self.__configDefaults[ 'shifterProxy' ] = ''
     self.__configDefaults[ 'shifterProxyLocation' ] = os.path.join( self.__configDefaults[ 'WorkDirectory' ],
                                                                         '.shifterCred' )
-
+    os.environ['AGENTNAME'] = agentName
```

Comment:
I have just seen that many agents define AGENT_NAME as a variable with a global scope. It is maybe a good idea to rationalize all this. Export it in the environment and set it as a global variable in the AgentModule does not seem unreasonable ?